plan/physical_plan_builder.go

```diff
@@ -912,15 +931,14 @@ func (p *Join) convert2PhysicalPlan(prop *requiredProperty) (*physicalPlanInfo,
 				break
 			}
 		}
-		if (p.preferINLJ&preferLeftAsOuter) > 0 && p.hasEqualConds() {
-			nljInfo, err := p.convert2IndexNestedLoopJoinLeft(prop, false)
-			if err != nil {
-				return nil, errors.Trace(err)
-			}
-			if nljInfo != nil {
-				info = nljInfo
-				break
-			}
+		forcedINLJ := (p.preferINLJ&preferLeftAsOuter) > 0 && p.hasEqualConds()
```

Comment:
We don't need to pass an argument to `p.convert2IndexNestedLoopJoinLeft`, it can be computed in that method.